data/alexandria start.txt

```diff
@@ -0,0 +1,1290 @@
+# Copyright (c) 2014 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Alexandria Library Card: Offer"
+	name "Alexandria Library Card"
+	minor
+	job
+	description "Alexandria Library is looking for an Apprentice Seeker. Meet Daemos Titan at the Al-Qarawiyyin Library on Earth to help him seek the Chal Jub Paqmey. Payments will total 400,000 credits. Escorts will be provided through dangerous areas."
+	to offer
+		not "library card"
+		"net worth" > 300000
+		has "event: remembrance day"
+		or
+			not "chosen sides"
+			has "main plot completed"
+	source
+		not planet Earth
+		near Sol 7
+	destination Earth
+	on complete
+		conversation
+			`You arrive on Earth and take a series of shuttles to get to Fez, Morocco where you expect to meet Daemos Titan. As with all old cities, the outer regions are monstrous mega-buildings built in the last few centuries. As you move inward, you see skyscrapers of the 20th to 22nd centuries.`
+			`	You turn a corner and see something you never thought you would see on the infamously-gray planet of Earth: a large, green, field of grass with a beautiful mosque. It looks like something out of Earth's ancient history, perfectly preserved in present day.`
+			choice
+				`	(Enter the library.)`
+			`On your way to the library, a scruffy-looking man approaches you. He has a rather large gun in his holster and a massive book strapped to his back. He starts staring intently at you, or possibly through you. His dark red sunglasses and porcupine beard make it difficult to read his face (or even see it). When the man abruptly stops walking you consider yelling for help, fearing you're about to be mugged or shot.`
```

Comment:
The style is that *every* line after the first line in a conversation received an additional tab within the backticks, not every line after the first line after a choice. This error exists throughout the missions.
```suggestion
			`	On your way to the library, a scruffy-looking man approaches you. He has a rather large gun in his holster and a massive book strapped to his back. He starts staring intently at you, or possibly through you. His dark red sunglasses and porcupine beard make it difficult to read his face (or even see it). When the man abruptly stops walking you consider yelling for help, fearing you're about to be mugged or shot.`
```

Reply:
I have corrected the indentation throughout the file so that the first paragraph the player sees (and no other) will be indented in each conversation.

There is one place where there are two possible first paragraphs, depending on branch logic. In that case both of the possible first paragraphs are indented. The player will only see one of the two.